app/models/state_file_base_intake.rb

```diff
@@ -46,9 +46,15 @@ def synchronize_df_dependents_to_database
     end
   end
 
+  def calculator
+    unless @calculator.present?
+      @calculator = tax_calculator
+      @calculator.calculate
+    end
+    @calculator
+  end
```

Comment:
This should cache the calculations in memory so that we don't accidentally rerun the calculations on the same web request.